src/fragments/lib/ios.mdx

```diff
@@ -10,6 +10,6 @@ This guide shows how to build an app using our Amplify Library for Swift and the
 
 <Callout>
 
-To upgrade from v1 to the new Amplify Library for Swift, you can follow our [upgrade guide here](https://github.com/aws-amplify/amplify-swift/blob/main/UpgradeGuide.md).
+To upgrade from v1 to the new Amplify Library for Swift, you can follow the [upgrade guide](/lib/project-setup/upgrade-guide/q/platform/ios/#structured-concurrency-asyncawait-pattern).
```

Comment:
```suggestion
To upgrade from v1 to the new Amplify Library for Swift, you can follow the [upgrade guide](/lib/project-setup/upgrade-guide).
```

We don't need the extra `/q/platform/ios/` part in the URL here - I think it causes an error when trying to click the link.

Also, what was the reason for linking directly to the header? Would it make more sense to just link directly to the page instead? As a user I was expecting it to take me to the top of the page where it had the overview of the upgrade guide and what changed since that's what the original link is describing

Reply:
@royjit, I also made a staging site for you here: https://royjit-upgradeguideswift.d3kzzkiugubm89.amplifyapp.com/lib/q/platform/ios/

Reply:
Thanks for the staging site. Accepted your suggestion as well.